docs/pages/admin-guides/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -132,11 +132,11 @@ teleport:
   #
   # We recommend to use tools like `pwgen` to generate sufficiently random
   # tokens of 32+ byte length.
-  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
+  # you can also use Auth Service instances IP, i.e. "10.1.1.10:3025"
```

Comment:
Problems with the botched comment aside, here I guess it'd be more correct to say "you can also use the IP of an Auth Service instance".